### ISSUE TITLE
Github language statistics are skewed by Jupyter Notebook files

### ISSUE THREAD

**Issue by philiplinden** [nyx-space/anise#183]
# Bug report
Jupyter Notebooks (`*.ipynb`) are large files with a lot of metadata. This causes the repository language stats to be dominated by Jupyter Notebook code when the majority of the project is not. ([Reference](https://towardsdatascience.com/how-to-ignore-jupyter-notebook-from-github-language-stats-6ec7004b885a))

Add the following line to the [.gitattributes](https://github.com/nyx-space/anise/blob/master/.gitattributes) to indicate to the stats tracker that this data is documentation, not code.

```
*.ipynb linguist-documentation=true
```

## Describe the bug
Jupyter Notebook metadata ends up being more the half of the code volume in the repository, but it is not half of the composition of the repository. The language stats should reflect the language of the product, not the docs.

## To Reproduce
1. Install [github-linguist](https://github.com/github-linguist/linguist) 
2. Run `github-linguist` in the root directory of this repo.
```
│ ~/repos/nyx-space/anise
└> github-linguist
76.80%  2450037    Jupyter Notebook
22.90%  730522     Rust
0.22%   6939       Python
0.07%   2166       Dhall
0.01%   260        Shell
0.00%   148        HTML
```

## Expected behavior
Rust should be listed as the majority of this project's code composition.

```
└> github-linguist                      
98.71%  730522     Rust
0.94%   6939       Python
0.29%   2166       Dhall
0.04%   260        Shell
0.02%   148        HTML
```

## Code to reproduce the issue
See above.

## Platform
GitHub

## Additional context
Linguist won't see changes to `.gitattributes` until you make a commit.

**Comment by ChristopherRabotin** [comment]
Hi Philip,

Thanks for the ticket and the proposed fix! I was definitely slightly
annoyed that my repo was marked as Jupyter notebook, so I'll implement this
in the morning.

Thanks!

**Comment by ChristopherRabotin** [comment]
Implement the fix in https://github.com/nyx-space/anise/commit/10a079329300ff6f9f61ffb64cbb4e3b00b664f0#diff-618cd5b83d62060ba3d027e314a21ceaf75d36067ff820db126642944145393e , will be merged with #182. I'll close this once the Github stats update the stats.

**Comment by ChristopherRabotin** [comment]
Fixed! Thank you!
